Editor/GradleWrapperSettings.cs

```diff
@@ -14,7 +14,7 @@ public static class GradleWrapperSettings
 
         public static string GradleVersion
         {
-            get => File.Exists(GRADLE_VERSION_FILE_PATH) ? File.ReadAllText(GRADLE_VERSION_FILE_PATH) : "";
+            get => File.Exists(GRADLE_VERSION_FILE_PATH) ? File.ReadAllText(GRADLE_VERSION_FILE_PATH).Trim() : "";
```

Comment:
Good job on the string trimming!
I hadn't though of many edge cases while developing the plugin, so this is a very nice addition.